app/models/case/ico/base.rb

```diff
@@ -258,7 +258,7 @@ def set_deadlines
     # For ICOs deadlines are manually set and don't need to be automatically
     # calculated. So this method called by a before_update hook in Case::Base
     # becomes a nop.
-    self.internal_deadline ||= @deadline_calculator.internal_deadline
+    nil
```

Comment:
This is set manually as the comment says, so this is not needed.
Looking at the history it appears to have been kept in temporarily during some changes, but never got removed.